scrunch/expressions.py

```diff
@@ -475,20 +489,37 @@ def category_ids(var_id, var_value, variables=variables):
                 return var_value
             return value
 
+        # special case for multiple_response variables
+        if len(subitems) == 2:
+            if 'value' in subitems[1] and 'variable' in subitems[0]:
+                var_url = subitems[0]['variable']
+                if ds.variables.index[var_url]['type'] == 'multiple_response':
+                    return adapt_multiple_response(ds, var_url, subitems[1]['value'])
+
         for item in subitems:
             if isinstance(item, dict) and 'variable' in item:
                 var_id = variable_id(item['variable'])
             elif isinstance(item, dict) and 'value' in item:
                 item['value'] = category_ids(var_id, item['value'])
             _subitems.append(item)
 
-        return _subitems
+        return _subitems, True
```

Comment:
What is this True? Why is it needed?

If we're returning an extra parameter, update the function's docstring to indicate what it means.